cli/actions/actions.go

```diff
@@ -27,6 +27,7 @@ type SedgeActions interface {
 	SetupContainers(SetupContainersOptions) error
 	RunContainers(RunContainersOptions) error
 	InstallDependencies(InstallDependenciesOptions) error
+	CreateJwtSecrets(CreateJwtSecretOptions) (string, error)
```

Comment:
```suggestion
	CreateJWTSecrets(CreateJwtSecretOptions) (string, error)
```
Plz, review the rest of the code to replace Jwt for JWT